sddi-base/Dockerfile

```diff
@@ -192,6 +192,11 @@ RUN set -ex && \
   ckan config-tool "${CKAN_INI}" "scheming.dataset_schemas = ckanext.scheming:ckan_dataset.yaml" && \
   ckan config-tool "${CKAN_INI}" "scheming.presets = ckanext.scheming:presets.json ckanext.repeating:presets.json ckanext.composite:presets.json" && \
   ckan config-tool "${CKAN_INI}" "scheming.dataset_fallback = false" && \
+  
```

Comment:
Is it necessary to ut these settings here. If these are the default values, this should be set upstream in the plugin. is it required to specify these config variables in `production.ini`?